src/plugins/pluginUtils.ts

```diff
@@ -22,16 +22,20 @@ export function simplifiedRegExp(input: undefined | string | RegExp): RegExp {
   if (!input) {
     return;
   }
+
   if (typeof input === 'string') {
     let r = '';
     for (let i = 0; i < input.length; i++) {
       switch (input[i]) {
         case '.':
-          r += '\\.';
+          r += '(\\.|.)';
```

Comment:
that doesn't make sense ;-) We need to escape all the periods, otherwise it will become a part of Regex

Reply:
fixed, we also need to test sparky better.
maybe manually spawing a child process and make it run a task ?